src/main.cpp

```diff
@@ -506,7 +510,7 @@ int main(int argc, char *argv[])
 		attroff(A_BOLD);
 
 		// Resolution adjustment status
-		if (!adjustResolution)
+		if (!adjustResolution || isCurrentAppDisabled)
```

Comment:
fwiw, don't think you needed this change here... `adjustResolution` is `!inDashboard && !isCurrentAppDisabled`, so `!adjustResolution || isCurrentAppDisabled` is effectively `inDashboard || isCurrentAppDisabled || isCurrentAppDisabled`.

Reply:
You're right, thanks!
fixed: https://github.com/Erimelowo/OpenVR-Dynamic-Resolution/commit/63693e0122628cc7cd5c94b8fcc1d98d6c41321b